src/components/simulator/result/SimResultChart.tsx

```diff
@@ -69,3 +52,26 @@ export const SimResultChart = ({ state, simulationType }: Props) => {
     </div>
   );
 };
+
+const Loading = () => (
+  <div className="flex h-[400px] items-center justify-center rounded-10 bg-black py-10">
+    <CarbonLogoLoading className="h-[100px]" />
+  </div>
+);
+
+const Error = ({ msg }: { msg?: string }) => {
+  return (
+    <div className="flex h-[400px] items-center justify-center rounded-10 bg-black py-10">
+      <div className="max-w-1/3 flex flex-col items-center justify-center space-y-10 rounded-10 border border-red bg-red/30 p-20">
+        <div className="text-24 font-weight-500">Error</div>
```

Comment:
This should be a heading.
```suggestion
        <h3 className="text-24 font-weight-500">Error</h3>
```
We'll need to have clean heading when we switch to the new font since we'll have a font dedicated for headings.